src/NewRecipe.css

```diff
@@ -0,0 +1,65 @@
+.New {
+  margin-bottom: 50px auto;
+  margin-top: 0px auto;
+  background-color: #282c34;
+  min-height: 100vh;
+  clear: both;
+  padding: 0;
```

Comment:
Default padding should be 0, so I don't think this is necessary. Happy to be wrong though.

Reply:
I guess it could inherit something, maybe you're accounting for that?